Add stInq Boss

WizarDude opened the pull request:
Steel Inquisitor boss added with Atium mechanic to mirror Vin's. Higher health and damage to counteract Vin's high starting Atium.

WizarDude committed:
Hero Work

Atium added, defence started and strike finished

WizarDude committed:
Update hero.py

WizarDude committed:
finished sim

WizarDude committed:
stInq added, attributes tweaked

The Steel Inquisitor boss fight was added, the Koloss were made a child of parent Enemy, and Vin's attributes were tweaked to handle the enemy buffs.

gwizzle651 commented:
im pretty sure that we are supposed to do the pr from a separate branch in our own repo

WizarDude committed:
Continue keys

Prevents barfing of entire program at once, now user confirmation required after every round

WizarDude committed:
roundCount added

Round ticker added for troubleshooting
